pyrsia_node/src/main.rs

```diff
@@ -26,7 +26,7 @@ extern crate warp;
 #[macro_use]
 extern crate lazy_static;
 //local module imports
-mod artifact_manager;
+pub mod artifact_manager;
```

Comment:
Main is not exposed 🤔 

```suggestion
mod artifact_manager;
```

Reply:
I do not understand this comment. 

The reason that I made this change is that without this I was getting MANY warning messages about functions in the artifact_manager not being called. By making artifact_manager public, the compiler is no longer sure that the functions are not being called and so the warnings stop.

Reply:
if there are more than 1 pub fn defined in module than we do not need to define pub mod

Reply:
@MitaliBo I'm sorry but I don't understand. What is the connection between whether a module if public and how many public functions it has?

Reply:
Nothing in a file called `main.rs` can be exposed so adding "pub" here does nothing.

Please remove it.

Reply:
@mgrand you mentioned about getting warnings in artifact_manager for functions not being called, so my comment was for that. Also it would have been better if artifact_manager followed the same protocol as other modules in the tree are following  , is to define a mod file and declare that module public in there and then just call in main, so that from main all the visibility part is hidden

Reply:
Done.